Makefile-daemon.am

```diff
@@ -1,11 +1,12 @@
 dbus_built_sources = rpm-ostreed-generated.h rpm-ostreed-generated.c
 
 rpm-ostreed-generated.h: rpm-ostreed-generated.c
-rpm-ostreed-generated.c: Makefile.am $(top_srcdir)/src/daemon/org.projectatomic.rpmostree1.xml
+rpm-ostreed-generated.c: Makefile $(top_srcdir)/src/daemon/org.projectatomic.rpmostree1.xml
 	$(AM_V_GEN) gdbus-codegen \
 		--interface-prefix org.projectatomic.rpmostree1 \
 		--c-namespace RPMOSTree \
 		--c-generate-object-manager \
+	  --c-generate-autocleanup=all \
```

Comment:
Kind of sucks to be dragged back by Fedora 24 😦...one of the major goals of rpm-ostree being making host system upgrades more reliable. But OK for now.